src/Tizen.Wearable.CircularUI.Forms.Renderer/CircleDateTimeSelectorRenderer.cs

```diff
@@ -102,14 +102,14 @@ void UpdateValueType()
         {
             if (null != Control && null != Element)
             {
+                Control.Format = "%d/%b/%Y%I:%M%p";
```

Comment:
`%I:%M%p` is not need on Date picker
And I think we provide locale preferred formatting

Reply:
done. remove %M%P in Date type.